### ISSUE TITLE
cannot build or test plugin

### ISSUE THREAD

**Issue by chrispy-snps** [dita-ot/org.dita.index#2]
## Expected Behavior
I am trying to build the plugin by running `./gradlew build`.

## Actual Behavior
I get various error messages, depending on which Java version I tried (8, 11, and 17).

## Steps to Reproduce
1. Run `rm -rf ~/.gradle` (just to rule it out as a problem).
2. Clone this repository and enter the directory.
3. Run `./gradlew build`.

## Copy of the error message, log file or stack trace
In Java 8, I get:

```
> Task :compileJava
.../git/org.dita.index/src/main/java/org/dita/index/configuration/ConfigEntryImpl.java:34: error: package com.google.common.annotations does not exist
import com.google.common.annotations.VisibleForTesting;
                                    ^
.../git/org.dita.index/src/main/java/org/dita/index/configuration/ConfigEntryImpl.java:39: error: cannot find symbol
@VisibleForTesting
 ^
  symbol: class VisibleForTesting
.../git/org.dita.index/src/main/java/org/dita/index/configuration/IndexConfiguration.java:37: error: cannot access XMLUtils
import org.dita.dost.util.XMLUtils;
                         ^
  bad class file: .../.gradle/caches/modules-2/files-2.1/org.dita-ot/dost/4.1.2/2355a00c6680e8d0a12c9fec1409fe477e3f33b5/dost-4.1.2.jar(org/dita/dost/util/XMLUtils.class)
    class file has wrong version 61.0, should be 52.0
    Please remove or make sure it appears in the correct subdirectory of the classpath.
```

I get other errors with Java 11 and Java 17.

## Environment
* Operating system and version: Linux (Ubuntu 20.04 LTS)
* Java version: 8, 11, 17

**Comment by jelovirt** [comment]
Fixed by #3

**Comment by chrispy-snps** [comment]
Thanks @jelovirt!
